### PR TITLE
chore(pom): adjust snapshot versions

### PR DESCRIPTION
**Pull request by tasso94**
related to camunda/camunda-bpm-platform#2348

### CODE REVIEW

File: bom/pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
   <groupId>org.camunda.spin</groupId>
   <artifactId>camunda-spin-bom</artifactId>
-  <version>1.23.0-SNAPSHOT</version>
+  <version>1.23.1-SNAPSHOT</version>
```
Comment (yanavasileva) [inline bom/pom.xml:14]: :x: As it's the `1.22`, the version should be `1.22.1` for all files.
```suggestion
  <version>1.22.1-SNAPSHOT</version>
```

**Review by yanavasileva — CHANGES_REQUESTED** [review]
Version must be `1.22.1-SNAPSHOT` as it's `1.22` branch.

File: bom/pom.xml (hunk 1 above)
Comment (tasso94) [reply bom/pom.xml]: Nice catch!

**Review by yanavasileva — APPROVED** [review]
👍 All good.

**Pull request merged**